manifests/jenkins.yaml

```diff
@@ -1,28 +1,36 @@
-# This is a fork of the `jenkins-ephemeral` OpenShift template because we need
+# This is a fork of the `jenkins-persistent` OpenShift template because we need
 # to be able to pass in more information to the Jenkins pod, such as env vars,
 # secrets, configmaps, etc...
 
-apiVersion: v1
+apiVersion: template.openshift.io/v1
 kind: Template
 labels:
   app: coreos-ci
   template: coreos-ci-jenkins-template
 metadata:
   annotations:
     description: |-
-      Jenkins service for CoreOS CI.
+      Jenkins service, with persistent storage.
+
+      NOTE: You must have persistent volumes available in your cluster to use this template.
     iconClass: icon-jenkins
-    openshift.io/display-name: CoreOS CI Jenkins
-    openshift.io/documentation-url: https://github.com/coreos/coreos-ci
-    openshift.io/support-url: https://github.com/coreos/coreos-ci
-    openshift.io/provider-display-name: CoreOS CI
+    openshift.io/display-name: Jenkins
+    openshift.io/documentation-url: https://docs.okd.io/latest/using_images/other_images/jenkins.html
+    openshift.io/long-description: This template deploys a Jenkins server capable
+      of managing OpenShift Pipeline builds and supporting OpenShift-based oauth login.
+    openshift.io/provider-display-name: Red Hat, Inc.
+    openshift.io/support-url: https://access.redhat.com
```

Comment:
Any reason we're dropping the CoreOS specific display name / URLs?

Reply:
Yeah, guess I could add those back. Though it's pretty meaningless in the end because we're not actually creating this template itself as an object, we're just instantiating directly from it at the CLI. Keeping it the same as upstream makes it easier to tell what the delta is when rebasing.

Reply:
I think he just missed it when updating. Along those lines it might be good to have one commit that is the exact jenkins persistent template (the .orig) and then a second commit which adds our customizations on top so it's easy to tell what we changed.